frontend/src/plugins/remote/auth/index.ts

```diff
@@ -197,6 +198,22 @@ class RemotePluginAuth {
     }
   }
 
+  /**
+   * Refreshes the current user infos, to fetch a new picture for instance
+   */
+  public async refreshCurrentUserInfo(): Promise<void> {
+    if (!isNil(this.currentUser) && !isNil(this.currentServer)) {
+      const api = useOneTimeAPI(
+        this.currentServer.PublicAddress,
+        this.getUserAccessToken(this.currentUser)
+      );
```

Comment:
If there's an user logged in, you don't need the `OneTimeApi`, you can use the global api instance. In fact if there's no global api defined, we can assume there's no current user/server. Check [this](https://github.com/jellyfin/jellyfin-vue/blob/master/frontend/src/plugins/remote/sdk/index.ts#L17-L51)

Reply:
Fair enough, this reminds me that I need to add a way to debug circular dependencies in Vite.